.idea/vcs.xml

```diff
@@ -13,4 +13,4 @@
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
-</project>
+</project>
```

Comment:
I don't believe you want to push such updates.

Reply:
done